src/octopeer-github/test/ChromeMock.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Created by Mitchell on 27-5-2016.
+ * Mocks chrome modules, functions and properties.
+ * Because of this mock, test cases can be performed on chrome dependent items (e.g. the storage).
+ */
+
+let _window: any = window;
+
+/* tslint:disable:no-empty */
```

Comment:
Youri said: why not `disable-next-line`? 
Mitchell said: will do this when moving the empty function to a function.